### PR TITLE
fix: [RGOeX-26170] html5 player - caption toggling and displaying for…

### PR DESCRIPTION
**Pull request by ihor-romaniuk**
**Description:** HTML5 player - caption toggling and displaying for Chrome and Safari browsers

**Youtrack:** https://youtrack.raccoongang.com/issue/RGOeX-26170

### CODE REVIEW

**Review by dyudyunov — APPROVED** [review]

File: video_xblock/backends/html5.py [hunk 1]
```diff
@@ -56,6 +56,8 @@ def get_frag(self, **context):
         """
         data_setup = Html5Player.player_data_setup(context)
         data_setup['sources'][0]['type'] = self.get_type(context['url'])
+        data_setup['html5'] = {'nativeTextTracks': False}
```
Comment (idegtiarov) [inline video_xblock/backends/html5.py:59]: @ihor-romaniuk Could we have any tests related to that change?

Comment (ihor-romaniuk) [reply video_xblock/backends/html5.py:59]: If we need to add some backend test, it's better to delegate this task to him.

Comment (dyudyunov) [reply video_xblock/backends/html5.py:59]: I think we should pass the task to the backend dev to cover Python changes but I don't know yet what to cover

Comment (idegtiarov) [reply video_xblock/backends/html5.py:59]: Will reassign the issue to @dyudyunov please clarify how we could test that code.

**Comment by dyudyunov** [comment]
rebased to resolve conflicts in CHANGELOG

File: video_xblock/backends/html5.py (hunk 1 above)
Comment (dyudyunov) [reply video_xblock/backends/html5.py:59]: Note for history:

Video xblock's tests are broken and should be fixed globally. We've decided to go without tests to not block the fix.

**Pull request merged**